.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: php
+
+php:
+    - 7.1
+    - 7.2
+
+before_script:
+    - wget http://getcomposer.org/composer.phar
```

Comment:
Why do you download it, since its included in default Php image?
https://docs.travis-ci.com/user/languages/php/#installing-composer-packages

Reply:
I copied it brainlessly from another of our project :dancer: Will remove it :)